examples/visualization/show_productspace.py

```diff
@@ -0,0 +1,48 @@
+# Copyright 2014, 2015 The ODL development group
+#
+# This file is part of ODL.
+#
+# ODL is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# ODL is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with ODL.  If not, see <http://www.gnu.org/licenses/>.
+
+""" Example on using show with ProductSpace's """
+
+# Imports for common Python 2/3 codebase
+from __future__ import print_function, division, absolute_import
+from future import standard_library
+standard_library.install_aliases()
+import odl
+import numpy as np
+
+n = 100
```

Comment:
>We're gathering tons of Pylint warnings with these names. I don't think names like n and m are necessarily bad, but here at least the m is a bit uncreative and lazy. A better name would be nfactors.

I agree.

>Generally, since we have legit uses of n and m, why don't we add them to the .pylintrc as accepted names?

[The list](https://github.com/odlgroup/odl/blob/e6fd57615387d0b8d7634876b7e12b16934cccd7/.pylintrc#L124) is huge, adding a few more wouldnt hurt :)